cli/cliutils/cliutils.go

```diff
@@ -327,6 +327,19 @@ func TrimOrg(org, id string) (string, string) {
 	return "", "" // will never get here
 }
 
+// Add the given org to the id if the id does not already contain an org
+func AddOrg(org, id string) string {
+	substrings := strings.Split(id, "/")
+	if len(substrings) <= 1 { // this means id was empty, or did not contain '/'
+		return fmt.Sprintf("%v/%v", org, id)
+	} else if len(substrings) == 2 {
+		return id
+	} else {
+		Fatal(CLI_INPUT_ERROR, i18n.GetMessagePrinter().Sprintf("the resource id can not contain more than 1 '/'"))
```

Comment:
Why does the message say "the resource id can not" instead of "the id can not". What is the word resource referring to?

Reply:
It is copied from the function TrimOrg() above the AddOrg().  TrimOrg is called in many places. The id passed in could be a pattern id, node id, service id. I guess the old code uses `resource id` as a general term. Shall I change "resource id" in both places to just "id"?

Reply:
So yes, I think we should remove the word resource from both places

Reply:
Done.